indigo_app/templates/indigo_app/place/_bulk_approve_form.html

```diff
@@ -19,13 +23,122 @@ <h5 class="modal-title">
         {% for work in works_in_progress %}
           <input type="hidden" name="works_in_progress" value="{{ work.pk }}">
         {% endfor %}
-        <div class="alert alert-warning mb-3">
+        {% for work in import_task_works %}
+          <input type="hidden" name="import_task_works" value="{{ work.pk }}">
+        {% endfor %}
+        {% for work in gazette_task_works %}
+          <input type="hidden" name="gazette_task_works" value="{{ work.pk }}">
+        {% endfor %}
+        {% for work, amendments in amendments_per_work.items %}
+          <input type="hidden" name="amendment_task_works" value="{{ work.pk }}">
+          {% for amendment in amendments %}
```

Comment:
the server can derive this information and the user isn't changing it (unless I'm missing something?) -- so it doesn't need to be passed to/from the form